services/static-webserver/client/source/class/osparc/po/Invitations.js

```diff
@@ -101,6 +101,13 @@ qx.Class.define("osparc.po.Invitations", {
       });
       form.add(userEmail, this.tr("User Email"));
 
+      const extraCredits = new qx.ui.form.Spinner().set({
```

Comment:
Suggestion: Try to make it optional (e.g. with a checkbox like the trial-account) and if not set, do not send any value. 
NOTE that the API exposes the extraCredits as an optional `PositiveInt` (i.e. does not include zero!)